DQM/SiPixelPhase1Heterogeneous/plugins/SiPixelPhase1MonitorRecHitsSoAGPU.cc

```diff
@@ -0,0 +1,116 @@
+// -*- C++ -*-
+///bookLayer
+// Package:    SiPixelPhase1MonitorRecHitsSoAGPU
+// Class:      SiPixelPhase1MonitorRecHitsSoAGPU
+//
+/**\class SiPixelPhase1MonitorRecHitsSoAGPU SiPixelPhase1MonitorRecHitsSoAGPU.cc 
+*/
+//
+// Author: Alessandro Rossi, Suvankar Roy Chowdhury
+//
+#include "HeterogeneousCore/CUDACore/interface/ScopedContext.h"
+#include "CUDADataFormats/TrackingRecHit/interface/TrackingRecHit2DHeterogeneous.h"
+// for string manipulations
+#include "DataFormats/SiPixelCluster/interface/SiPixelCluster.h"
+#include "Geometry/CommonDetUnit/interface/PixelGeomDetUnit.h"
+#include "Geometry/CommonTopologies/interface/PixelTopology.h"
+#include "DataFormats/SiPixelDetId/interface/PixelSubdetector.h"
+#include "SiPixelPhase1MonitorRecHitsSoABase.h"
+
+class SiPixelPhase1MonitorRecHitsSoAGPU : public SiPixelPhase1MonitorRecHitsSoABase {
+public:
+  explicit SiPixelPhase1MonitorRecHitsSoAGPU(const edm::ParameterSet&);
+  ~SiPixelPhase1MonitorRecHitsSoAGPU() override = default;
+  void analyze(const edm::Event& iEvent, const edm::EventSetup& iSetup) override;
+  static void fillDescriptions(edm::ConfigurationDescriptions& descriptions);
+
+private:
+  edm::EDGetTokenT<cms::cuda::Product<TrackingRecHit2DGPU>> tokenHitsGPU_;
+  edm::EDGetTokenT<SiPixelClusterCollectionNew> clusterToken_;
```

Comment:
```suggestion
  const edm::EDGetTokenT<cms::cuda::Product<TrackingRecHit2DGPU>> tokenHitsGPU_;
  const edm::EDGetTokenT<SiPixelClusterCollectionNew> clusterToken_;
```

if they're put into the initializer's list at line 40-41.

Reply:
also the convention for the member data is inconsistent (either use leading `m_` or trailing `_`)